includes/ucf-degree-search-api.php

```diff
@@ -246,19 +246,43 @@ public static function prepare_degree_for_response( $post, $request ) {
 		$hours = get_post_meta( $post->ID, 'degree_hours', true );
 		$terms = wp_get_post_terms( $post->ID, 'program_types' );
 		$term = is_array( $terms ) ? $terms[0]->slug : null;
+		$colleges = self::get_college_terms( $post, $request );
 
 		$retval = array(
 			'title'     => $post->post_title,
 			'nameShort' => $name_short,
 			'url'       => $permalink,
 			'hours'     => $hours,
 			'type'      => $term,
+			'colleges'  => $colleges,
 			'subplans'  => array()
 		);
 
 		return $retval;
 	}
 
+	/**
+	 * Retrieves a simplified array of college terms
+	 *
+	 * @param WP_Post $post The post object
+	 * @param WP_REST_Request $request The request object
+	 * @return array The array of colleges.
+	 */
+	public static function get_college_terms( $post, $request ) {
+		$colleges = wp_get_post_terms( $post->ID, 'colleges' );
```

Comment:
`$colleges` will return a WP_Error object here if the colleges taxonomy isn't defined; since that taxonomy isn't assumed to be installed alongside this plugin, we'll need to add a check to make sure `$colleges` is actually an array before attempting to loop through it on line 276